src/Commands/AddCommand.php

```diff
@@ -35,6 +35,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $addedHooks = [];
         $gitDir = $input->getOption('git-dir');
+        $hookDir = "{$gitDir}/hooks";
+
+        if (! is_dir($hookDir)) {
+            mkdir($hookDir);
```

Comment:
This would fail if the `gitDir` folder does not already exist. We need to make the `mkdir` recursive. Do you mind fixing that? You can get more info [here](http://php.net/mkdir). Or you can look at how the git hooks folders were created in the `AddCommand` and `UpdateCommand` tests.